proposals/b:zoe1/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Thanks for catching that. The `.gitignore` does ignore it, but the pattern assumes `.yarn` dir will only be at root. https://github.com/yarnpkg/berry/blob/master/.gitignore

I could change the ignore but WDYT of having one yarn.lock for the repo and making all the other packages workspaces? I've shied away because under the existing structure the root package would have to be in `/usr/src` of the Docker image, which is icky. I could refactor so that `proposals` is under `upgrade-test-scripts`, next to `lib`, and then `upgrade-test-scripts` has the one root. Though it would still require changing the gitignore

Reply:
one yarn.lock for the repo sounds attractive.

I am, as usual, reticent to move things around.

I guess don't feel strongly one way or the other. This repo is still young.